src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+    <include resource="org/springframework/boot/logging/logback/base.xml"/>
+    <logger name="*.com.synopsys.*" level="trace"/>
+    <logger name="org.springframework.core " level="error"/>
+    <logger name="org.springframework.beans" level="error"/>
+    <logger name="org.springframework.context" level="error"/>
+    <logger name="org.springframework.transaction" level="error"/>
+    <logger name="org.springframework.web" level="error"/>
+    <logger name="org.springframework.test" level="error"/>
+    <logger name="org.hibernate" level="error"/>
+</configuration>
```

Comment:
Added this new file to stop our test logs from being spammed with useless information. We will now see errors and all the things we log at Synopsys.